README.md

```diff
@@ -53,18 +53,17 @@ chmod +x minio
 ```
 
 ### Snap
-You can install the latest `minio` [snap](https://snapcraft.io), and help testing the most recent changes of the master branch in [all the supported Linux distros](https://snapcraft.io/docs/core/install) with:
+You can install the latest `minio` [snap](https://snapcraft.io).
 
 ```sh
 sudo snap install minio --edge
 ```
 
-Every time a new version of `minio` is pushed to the store, you will get it updated automatically.
-
-You will need to allow the minio snap to observe mounts in the system:
+Every time a new version of `minio` is pushed to the store, you will get it updated automatically. You will need to allow the minio snap to observe mounts in the system:
```

Comment:
For auto updates to minio binary from the snap store, do :

Reply:
Actually the original text is confusing, snap auto updates any local installation.

Reply:
Yea. However 'you will get it updated automatically' sounds weird. We should say it more straightforward like : ' Every time a new version of minio is published at the store, your minio instance is auto updated. Please allow minio snap to observe mounts in your system.

Reply:
I have an issue with these two parts :  
 * you will get *it* updated automatically. - Ambiguous what it is.
 * allow minio snap *to* observe mounts. - It can be allow to if thats exactly how its expressed. If not we could say it  differently such as :  Configure minio snap as follows :